frontend/packages/dev-console/integration-tests/support/pages/app.ts

```diff
@@ -259,9 +268,28 @@ export const yamlEditor = {
       .clear();
   },
 
+  replaceContentInYamlEditor: (src: string, target: string) => {
+    cy.get(yamlPO.yamlEditor)
+      .click()
+      .focused()
+      .type('{ctrl}h');
+    cy.get(yamlPO.yamlEditor).within(() => {
+      cy.get('textarea[placeholder="Find"]').type(src);
+      cy.get('textarea[placeholder="Replace"]').type(target);
+    });
```

Comment:
@makambalaji As discussed over the call, we need to move away from the os agnostic commands to replace the content to avoid the failure in macos and in CI as well. 

Thanks @invincibleJai for sharing an alternate way to create a new revision, to avoid the above issue.

Reply:
> Based on your last [commit](https://github.com/openshift/console/compare/0c4c32fa75d889c2918450494359e41d04cd21ee..c4f9702dc0ebf06f984f06412b01bfc49ba8ff02) to fix the test failures, Your README.md file changes got reverted, Is it intentional ?
> 
> Current folder structure info and test command (removed in last [commit](https://github.com/openshift/console/compare/0c4c32fa75d889c2918450494359e41d04cd21ee..c4f9702dc0ebf06f984f06412b01bfc49ba8ff02#diff-be9b118465ff86a203f476dc4e9986b54650bcf5cdeb1e030d4b0eac929effcc)):
> 
> ```
> |   ├──  e2e               <--- Gherkin scenarios executing on CI
> |   |    └──knative-ci.feature
> |   ├──  eventing           <--- Eventing gherkin scenarios
> ```

I will update it